gluoncv/model_zoo/mask_rcnn/mask_rcnn.py

```diff
@@ -173,8 +173,9 @@ def __init__(self, features, top_features, classes, mask_channels=256, rcnn_max_
                 roi_size = (self._roi_size[0] * 2, self._roi_size[1] * 2)
             else:
                 roi_size = self._roi_size
+            self._roi_size = roi_size
             self.mask_target = MaskTargetGenerator(
-                self._max_batch, self._num_sample, self.num_class, roi_size)
+                self._max_batch, self._num_sample, self.num_class, self._roi_size)
```

Comment:
the roi size here is actually different than roi size used in other places thus it is not assigned to self._roi_size. maybe a field is needed. When deep_fcn is used with FPN, it upsamples the output size to 2x the original typically 28x28, where as the default does not use FPN, which keeps the original 14x14 size.